### PR TITLE
[ETP-820] Add template tag to properly check if The Events Calendar is active.

### PR DESCRIPTION
**Pull request by juanfra**
### 🎫 Ticket

[ETP-820] 

### 🗒️ Description

- Add a template tag function to properly check if The Events Calendar is active.

### ✔️ Checklist
- [ ] I've included a changelog entry in the readme.txt file. 
- [ ] My code is tested. 
- [ ] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ETP-820]: https://theeventscalendar.atlassian.net/browse/ETP-820?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/template-tags/tickets.php [hunk 1]
```diff
@@ -1834,4 +1834,18 @@ function tribe_tickets_get_provider_query_slug() {
...
+ * Return if `The Events Calendar` is active.
+ *
+ * By checking if `tribe_events` function exists we avoid fatal errors when there are
+ * version dependencies mismatches.
+ *
+ * @since TBD
+ *
+ * @return bool True if The Events Calendar is active.
+ */
+function tec_tickets_tec_events_is_active() {
```
Comment (rafsuntaskin) [inline src/template-tags/tickets.php:1849]:
```suggestion
function tec_tickets_tec_events_is_active() : bool {
```

Comment (rafsuntaskin) [reply src/template-tags/tickets.php:1849]: we may add the new format.

Comment (juanfra) [reply src/template-tags/tickets.php:1849]: ok

**Review by rafsuntaskin — APPROVED** [review]
🥇

**Pull request merged**